acceptance/testdata/repos/repo-fork-sync.txtar

```diff
@@ -0,0 +1,44 @@
+# Use gh as a credential helper
+exec gh auth setup-git
+
+# Create and clone a repository with a file so it has a default branch
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private --clone
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Fork and clone the repo
+exec gh repo fork $ORG/$SCRIPT_NAME-$RANDOM_STRING --org $ORG --fork-name $SCRIPT_NAME-$RANDOM_STRING-fork --clone
+
+# Defer fork cleanup
+defer gh repo delete $ORG/$SCRIPT_NAME-$RANDOM_STRING-fork --yes
+
+# Sleep so that the BE has time to sync
+sleep 5
```

Comment:
I would absolutely love that! I had briefly thought about some sort of polling solution in place of `sleep`, but thought that since `sleep` got the job done, here, I'd punt it until later. As such, this is right in line with what I would expect, here 👍

Reply:
Yes totally 100% let's keep `sleep`. There are some edge cases to figure out around the other one, I just wanted to float it to get your opinion.